src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+    private static final int ARREY_SIZE = 16;
```

Comment:
There is a typo in the constant name `ARREY_SIZE`. It should be `ARRAY_SIZE` to correctly reflect the intended meaning.